src/sentry/api/endpoints/organization_member_unreleased_commits.py

```diff
@@ -43,7 +45,13 @@ def get(self, request, organization, member):
             is_verified=True,
         ).values_list('email', flat=True))
         if not email_list:
-            return self.respond([])
+            return self.respond({
```

Comment:
ideally this would also expand with more errors for if we didnt find any CommitAuthor's matching your username as well as not having any released repos (so no commits even possible)